package.json

```diff
@@ -24,9 +24,9 @@
     "configure": "node-gyp configure",
     "build": "node-gyp build",
     "install": "node-gyp rebuild",
-    "concourse-dependencies": "make info && make electron-develop",
     "concourse-test": "npm test",
-    "concourse-build-installers": "make installers-all"
+    "concourse-test-electron": "npm test",
```

Comment:
Is there electron test not linting & sanity checking?

Reply:
The `npm test` script does all that already

Reply:
Yeah, the difference is `npm test` does `make test` vs `make test-cli` below